Mathlib/LinearAlgebra/AffineSpace/AffineSubspace.lean

```diff
@@ -1607,6 +1642,33 @@ end AffineMap
 
 namespace AffineEquiv
 
+section ofEq
+variable (S₁ S₂ : AffineSubspace k P₁) [Nonempty S₁] [Nonempty S₂]
+
+attribute [local instance] AffineSubspace.toAddTorsor
+
+/-- Affine equivalence between two equal affine subspace.
+
+This is the affine version of `LinearEquiv.ofEq`. -/
+@[simps linear]
+def ofEq (h : S₁ = S₂) : S₁ ≃ᵃ[k] S₂ where
```

Comment:
Here the existing names are more chaotic, and `ofEq` is a reasonable choice:
[Equiv.Set.ofEq](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Logic/Equiv/Set.html#Equiv.Set.ofEq), [RingEquiv.subringCongr](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Subring/Basic.html#RingEquiv.subringCongr), [IntermediateField.equivOfEq](https://leanprover-community.github.io/mathlib4_docs/Mathlib/FieldTheory/Adjoin.html#IntermediateField.equivOfEq), [Subalgebra.equivOfEq](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Algebra/Subalgebra/Basic.html#Subalgebra.equivOfEq)